compiler/rustc_mir/src/dataflow/framework/graphviz.rs

```diff
@@ -570,6 +571,13 @@ where
     }
 }
 
+macro_rules! regex {
+    ($re:literal $(,)?) => {{
+        static RE: SyncOnceCell<regex::Regex> = SyncOnceCell::new();
+        RE.get_or_init(|| Regex::new($re).unwrap())
```

Comment:
It seems like this should use `SyncLazy` instead of `SyncOnceCell`. @matklad is using `Lazy` preferable to `OnceCell` when possible?

Reply:
Depends on code-style I guess. My preference would be to use `OnceCell` here, rather than lazy. I generally like to use the smallest tool for the job, and, as we are not really going to use `Lazy`'s deref in any notable capacity here, `OnceCell` seems preferable. 

But I also can see "don't overthink this and use just `Lazy`" line of thought there :-)